types/index.d.ts

```diff
@@ -333,7 +333,8 @@ export namespace Operation {
   interface AllowTrust extends BaseOperation<OperationType.AllowTrust> {
     trustor: string;
     assetCode: string;
-    authorize: boolean | undefined;
+    // this is a boolean or a number so that it can support protocol 12 or 13
+    authorize: boolean | number | undefined;
```

Comment:
@ire-and-curses @tamirms I'll actually do the same for this, add the breaking change, making it a number.